demo/index.html

```diff
@@ -10,47 +10,80 @@
 <!doctype html>
 <html>
 <head>
-  <title>paper-progress demo</title>
+  <title>paper-linear-progress demo</title>
 
-  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <meta name="viewport"
+        content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-capable" content="yes">
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <!-- Polyfill promises if needed -->
+  <script>
+    var script;
+    if (typeof Promise != 'function' || typeof Promise.prototype.then != 'function') {
+      script = document.createElement('script');
+      script.src = '../../es6-promise/promise.js';
+      document.head.appendChild(script);
+    }
+  </script>
```

Comment:
Building off of the note here https://github.com/PolymerElements/paper-linear-progress/pull/1/files#r46194828...

Once you have updated the dependency, you can bring in the polyfill with:

```html
<link rel="import" href="../../promise-polyfill/promise-polyfill.html">
```